cluster-diagnosis/sysdumpcollector.py

```diff
@@ -260,31 +263,32 @@ def collect_cilium_bugtool_output(self, label_selector):
     def collect_cilium_bugtool_output_per_pod(self, podstatus):
         bugtool_output_file_name = "bugtool-{}-{}.tar".format(
             podstatus[0], utils.get_current_time())
-        cmd = "kubectl exec -n kube-system {} cilium-bugtool".format(
-            podstatus[0])
+        cmd = "kubectl exec -n {}, {} cilium-bugtool".format(
+            namespace.name, podstatus[0])
         try:
             encoded_output = subprocess.check_output(cmd.split(), shell=False)
         except subprocess.CalledProcessError as exc:
             if exc.returncode != 0:
                 log.error(
                     "Error: {}. Could not run cilium-bugtool on {}"
                     .format(exc, podstatus[0]))
-        else:
-            output = encoded_output.decode()
-            p = re.compile(
-                "^ARCHIVE at (.*)$")
-            output_file_name = ""
-            for line in output.splitlines():
-                match = p.search(line)
-                if match:
-                    output_file_name = match.group(1)
-            if output_file_name == "":
-                log.error(
-                    "Error: {}. Could not find cilium-bugtool output"
-                    " file name".format(exc))
+            else:
```

Comment:
This indentation change is incorrect. The else portion is for the try block (execute this if there is no exception).

Moreover, this change leads to the following error now:
```
 python cluster-diagnosis.zip sysdump
INFO collecting kubernetes version info ...
INFO collected kubernetes version info: k8s-version-info-20181114-090256.txt
INFO collecting nodes overview ...
INFO collected nodes overview: nodes-20181114-090256.json
INFO collecting pods overview ...
INFO collected pods overview: pods-20181114-090256.json
INFO collecting pods summary ...
INFO collected pods summary: pods-20181114-090257.txt
INFO collecting services overview ...
INFO collected svc overview: services-20181114-090257.yaml
INFO collecting cilium gops stats ...
INFO collected gops stats file: cilium-wr947-20181114-090257-stats.txt
INFO collected gops memstats file: cilium-wr947-20181114-090257-memstats.txt
INFO collected gops stack file: cilium-wr947-20181114-090258-stack.txt
INFO collecting cilium network policy ...
INFO collected cilium network policy: cnp-20181114-090258.yaml
INFO collecting cilium etcd secret ...
Error from server (NotFound): secrets "cilium-etcd-secrets" not found
ERROR Error: Command 'kubectl get secret cilium-etcd-secrets -n kube-system -o json' returned non-zero exit status 1. Unable to get and redact cilium secret
INFO collecting cilium endpoints ...
INFO collected cilium endpoints: cep-20181114-090258.yaml
INFO collecting cilium daemonset yaml ...
INFO collected cilium daemonset yaml file: cilium-ds-20181114-090259.yaml
INFO collecting cilium configmap yaml ...
INFO collected cilium configmap yaml file: cilium-configmap-20181114-090259.yaml
INFO collecting cilium-bugtool output ...
Error from server (NotFound): pods "cilium-wr947" not found
ERROR Error: Command '['kubectl', 'exec', '-n', 'kube-system,', 'cilium-wr947', 'cilium-bugtool']' returned non-zero exit status 1. Could not run cilium-bugtool on cilium-wr947
Traceback (most recent call last):
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/runpy.py", line 162, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "cluster-diagnosis.zip/__main__.py", line 88, in <module>
  File "cluster-diagnosis.zip/sysdumpcollector.py", line 359, in collect
  File "cluster-diagnosis.zip/sysdumpcollector.py", line 259, in collect_cilium_bugtool_output
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/multiprocessing/pool.py", line 251, in map
    return self.map_async(func, iterable, chunksize).get()
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/multiprocessing/pool.py", line 567, in get
    raise self._value
UnboundLocalError: local variable 'output_file_name' referenced before assignment
```